mempool/reactor.go

```diff
@@ -131,13 +144,26 @@ func (memR *MempoolReactor) GetChannels() []*p2p.ChannelDescriptor {
 // AddPeer implements Reactor.
 // It starts a broadcast routine ensuring all txs are forwarded to the given peer.
 func (memR *MempoolReactor) AddPeer(peer p2p.Peer) {
-	memR.ids.ReserveForPeer(peer)
+	_ = memR.ids.ReserveForPeer(peer)
 	go memR.broadcastTxRoutine(peer)
 }
 
 // RemovePeer implements Reactor.
 func (memR *MempoolReactor) RemovePeer(peer p2p.Peer, reason interface{}) {
 	memR.ids.Reclaim(peer)
+
+	// XXX (melekes): commented out for performance reasons
```

Comment:
It's not clear why this is added but immediately commented out. OK, it says performance reasons but why is the commented code added in the first place then?